core/roboconf-messaging-rabbitmq/src/main/java/net/roboconf/messaging/rabbitmq/DmReturnListener.java

```diff
@@ -67,12 +67,7 @@ public void handleReturn(
 			Utils.logException( this.logger, e );
 		}
 
-		StringBuilder sb = new StringBuilder();
-		sb.append( "A message sent by the DM was not received by any agent queue." );
-		sb.append( "\nMessage type: " + messageType );
-		sb.append( "\nRouting key: " + routingKey );
-		sb.append( "\nReason: " + replyText );
-
-		this.logger.warning( sb.toString());
+		this.logger.warning("A message sent by the DM was not received by any agent queue." + "\nMessage type: "
```

Comment:
Please, use a string builder.
And before adding content to the string builder, check that the logger level is compatible with the WARNING one.

Otherwie, we will concatenate strings and if we are in SEVERE, we will drop the string.